src/components/transaction-list/index.jsx

```diff
@@ -62,9 +63,23 @@ export const TransactionList = ({ primary, count, className }) => {
       ) }
     >
       {
-        transactions.map((item, index) => (
+        transactions && groupDate(transactions).map((item, index) => (
           <div key={ index } className={ style.transactionsGroup }>
-            <Text size='small' className={ style.transactionsHeader }>{ item.date }</Text>
+            <Text
+              size='small'
+              className={ style.transactionsHeader }
+            >
+              {
+                dayjs(item.date).calendar(null, {
+                  sameDay: `[${t('transactions.calendar.sameDay')}], DD.MM`,
+                  nextDay: 'dddd DD.MM',
+                  nextWeek: 'dddd DD.MM',
+                  lastDay: 'dddd, DD.MM',
+                  lastWeek: 'dddd, DD.MM',
+                  sameElse: 'dddd DD.MM',
```

Comment:
Why some strings contain coma and other don't?

Reply:
Bugs fixed